src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy

```diff
@@ -96,7 +97,10 @@ class Server42FlowRttSpec extends HealthCheckSpecification {
         }
 
         cleanup: "Revert system to original state"
-        revertToOrigin([flow], flowRttFeatureStartState, initialSwitchRtt)
+        flow && flowHelperV2.deleteFlow(flow.flowId) && switchHelper.verifyAbsenceOfServer42FlowRttRules(initialSwitchProps.keySet())
```

Comment:
Due to the specificity of the test(verifyAbsenceOfServer42FlowRttRules), flow deletion should be executed as a first step of the cleanup section.